ESLint-Formatter.py

```diff
@@ -38,6 +38,9 @@ def run(self, edit):
 
     output = self.run_script_on_file(self.view.file_name())
 
+    # log output for easier debugging
```

Comment:
I can look into only printing it when debug mode for SublimeLinter is on. IMO it'll be good to have as the plugin was failing on me silently (with debug mode on) which made it harder to debug the issue.